backend/src/main/java/com/woowacourse/naepyeon/repository/rollingpaper/RollingpaperRepositoryImpl.java

```diff
@@ -39,11 +39,16 @@ public Page<Rollingpaper> findByAddresseeId(final Long addresseeId, final Pageab
     }
 
     @Override
-    public List<Rollingpaper> findByTeamId(final Long teamId) {
-        return queryFactory
+    public List<Rollingpaper> findByTeamIdAndOrder(final Long teamId, final String order) {
+        final JPAQuery<Rollingpaper> jpaQuery = queryFactory
                 .selectFrom(rollingpaper)
-                .where(isTeamIdEq(teamId))
-                .fetch();
+                .where(isTeamIdEq(teamId));
+        if (order.equals("oldest")) {
+            return jpaQuery.orderBy(rollingpaper.createdDate.desc())
+                    .fetch();
+        }
+        // oldest 외의 파라미터 값은 전부 최신 순으로 리턴
+        return jpaQuery.fetch();
```

Comment:
```suggestion
        return queryFactory
                .selectFrom(rollingpaper)
                .where(isTeamIdEq(teamId))
                .orderBy(orderByCreateDate(order))
                .fetch();
    }

    private OrderSpecifier<LocalDateTime> orderByCreateDate(final String order) {
        if (order.equals("oldest")) {
            return new OrderSpecifier<>(Order.DESC, rollingpaper.createdDate);
        }
        return new OrderSpecifier<>(Order.ASC, rollingpaper.createdDate);
    }
```

OrderSpecifier에 대해서 공부중이지만 이런식으로 해볼 수 있지 않을까요 ?

Reply:
이 방법도 신선하네요 👍 
정렬 요건 처리를 서비스 쪽으로 위임하면서 querydsl로 두 개의 메서드로 나뉘는 방안으로 수정됐습니다.